src/tools/rbd/action/BenchWrite.cc

```diff
@@ -262,6 +262,11 @@ int execute(const po::variables_map &vm) {
   } else {
     bench_io_size = 4096;
   }
+  if ((bench_io_size > 100*1024*1024) || (bench_io_size < 1))
```

Comment:
Hi @x11562 , I am curious how dose the "too large" exactly mean. 100M? or 1G? is that a limit in buffer creating? I think only when we find out the root cause of the crash, we can set a better limit here.